internal/impl/deploy.go

```diff
@@ -86,6 +88,16 @@ func Deploy(ctx context.Context, configFilename string) error {
 	if config.Repo == "" {
 		fmt.Fprintln(os.Stderr, "No container repo specified in the config file. The container image will only be accessible locally. See `weaver kube deploy --help` for details.")
 	}
+	switch config.BuildTool {
+	case "docker":
```

Comment:
if config.BuildToold is already "docker" or "podman" why do we need to set it again?

```go
switch config.BuildTool {
   case "docker", "podman": // do nothing
   case "": // not specified
      config.BuildTool = defaultBuildTool
   default:
      return ...
}```

Reply:
oh true, let me fix that.